docs/pages/modules/use-standalone-expo-module-in-your-project.mdx

```diff
@@ -119,52 +139,58 @@ Open another terminal window, compile and run the example app:
+<Step label="3">
+
+### Publish the package to npm
+
+To publish your package to npm, you need an npm account. If you don't have one, create an account on [the npm website](https://www.npmjs.com/signup). After creating an account, log in by running the following command:
 
 <Terminal cmd={['$ npm login']} />
 
-Then, navigate to the root of your module and publish your module running:
+Navigate to the root of your module, then run the following command to publish it:
 
 <Terminal cmd={['$ npm publish']} />
 
 Your module will now be published to npm and can be installed in other projects using `npm install`.
 
-Apart from publishing your module to npm, there are other ways to use it in your project:
+Apart from publishing your module to npm, you can use it in your project in the following ways:
+
+- **Create a tarball**: Use `npm pack` to create a tarball of your module, then install it in your project by running `npm install /path/to/tarball`. This method is helpful for testing your module locally before publishing it or sharing it with others who don't have access to the npm registry.
+- **Run a local npm registry**: Use a tool like [Verdaccio](https://verdaccio.org/) to host a local npm registry. You can install your module from this registry, which is useful for managing internal packages within a company or organization.
+- **Publish a private package**: [Use a private registry with EAS Builds](/build-reference/private-npm-packages/) to manage private modules securely.
 
-- Create a tarball of your module using `npm pack` and install it directly in your project by running `npm install /path/to/tarball`. It's useful to test your module locally before publishing it to npm or if you want to share it with others who don't have access to the npm registry.
-- Run a local npm registry, such as [Verdaccio](https://verdaccio.org/), and install your module from there. This is useful for cases where you want to host your private npm registry to manage internal packages within a company or organization.
-- [Publish a private package or use a private registry with EAS Builds](/build-reference/private-npm-packages/).
+</Step>
 
-### 4. Test the published module
+<Step label="4">
 
-To test the published module in a new project, create a new app and install the module as a dependency:
+### Test the published module
+
+To test the published module in a new project, create a new app and install the module as a dependency by running the following command:
 
 <Terminal
   cmdCopy="npx create-expo-app my-app && cd my-app && npx expo install expo-settings"
   cmd={['$ npx create-expo-app my-app', '$ cd my-app', '$ npx expo install expo-settings']}
 />
 
-You should now be able to use the module inside your app! To test it, edit the **App.js** in the app and render the text message from **expo-settings**.
+You can now use the module in your app! To test it, edit **App.js** and render the text message from **expo-settings**.
```

Comment:
Again, creating a new app here, the default will be used and the entry point file will be **app/(tabs)/index.tsx**. Can you please update that here and in the previous section?